examples/alpaka/nbody/nbody.cpp

```diff
@@ -64,9 +64,9 @@ using Particle = llama::Record<
 
 enum Mapping
 {
-    AoS,
-    SoA,
-    AoSoA
+    ao_s,
+    so_a,
+    ao_so_a
```

Comment:
alpaka VC: we should either add exceptions for these, since they are terms of art, or manually rename then to lower case identifiers (which should not trigger the check).